docs/cli/reference.md

```
@@ -618,6 +620,27 @@ Restore a trashed session, returning it to its prior bucket with its transcript
 
 
 
+## `aoe session import`
+
+Import existing Claude Code sessions from disk. Scans the given path(s) (default: current directory) for Claude Code conversations whose working directory is at or under a path, and creates an AoE session for each: a terminal/tmux session that resumes the conversation with `claude --resume <id>` (default), or a structured-view session with `--structured`
+
+**Usage:** `aoe session import [OPTIONS] [PATHS]...`
+
+###### **Arguments:**
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h6

(MD001, heading-increment)